msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AuthenticationErrorCode.java

```diff
@@ -115,4 +115,12 @@ public class AuthenticationErrorCode {
      * A JWT parsing failure, indicating the JWT provided to MSAL is of invalid format.
      */
     public final static String INVALID_JWT = "invalid_jwt";
+    /**
+     * Indicates that a Broker implementation is missing from the device, such as when an app developer
+     * does not include one of our broker packages as a dependency in their project, or otherwise cannot
+     * be accessed by MSAL Java
+     *
+     * TODO: link to repository wiki page describing this more and how to fix
```

Comment:
TODO should be resolved

Reply:
Removed TODO. Added a task in ADO to take care of this.